README.md

```diff
@@ -1,9 +1,117 @@
 
-The Unbiased API server can be configured using environment variables. The following environment variables need to be set for the server to start:
+## Why choose **unbiased**?
+
+Our goal in creating **unbiased** was to provide a user-friendly yet powerful tool that addresses the nuanced demands of clinical trial randomization. It offers:
+
+- **Ease of Integration**: Designed to fit effortlessly into your research workflow.
+- **Adaptability**: Whether for small-scale studies or large, multi-center trials, **unbiased** scales to meet your needs.
+- **Comprehensive Documentation**: To support you in applying the package effectively.
+
+By choosing **unbiased**, you're adopting a sophisticated approach to trial randomization, ensuring fair and efficient participant allocation across your studies.
+
+## Core features
+
+The **unbiased** package integrates dynamic and traditional randomization methods, including:
+
+- **Minimization Method**: For balanced allocation considering covariates.
+- **Simple Randomization**: For straightforward, unbiased participant assignment.
+- **Block Randomization**: To ensure equal group sizes throughout the trial.
+
+Available both as a standard R package and through an API, **unbiased** provides flexibility for researchers. It ensures seamless integration with electronic Case Report Form (eCRF) systems, facilitating efficient patient management.
+
+## Table of Contents
+
+1. [Background](#background)
+   - [Purpose and Scope for Clinical Trial Randomization](#purpose-and-scope-for-clinical-trial-randomization)
+   - [Comparative Analysis of Randomization Methods](#comparative-analysis-of-randomization-methods)
+   - [Comparison with Other Solutions](#comparison-with-other-solutions)
+2. [Quickstart Guide](#quickstart-guide)
+     - [Installation Instructions](#installation-instructions)
+     - [Deploying the API](#deploying-the-api)
+     - [API Configuration](#api-configuration)
+3. [Getting started with **unbiased**](#getting-started-with-unbiased)
+     - [Using Randomization Functions within R](#using-randomization-functions-within-r)
+         - [Simple Randomization](#simple-randomization)
+         - [Minimization Method](#minimization-method)
+     - [API Endpoints](#api-endpoints)
+        - [Study Creation](#study-creation)
+        - [Patient Randomization](#patient-randomization)
+4. [Technical Implementation](#technical-implementation)
+   - [Quality Assurance Measures](#quality-assurance-measures)
+   - [Running Tests](#running-tests)
+     - [Executing Tests from an R Interactive Session](#executing-tests-from-an-r-interactive-session)
+     - [Executing Tests from the Command Line](#executing-tests-from-the-command-line)
+     - [Running Tests with Docker Compose](#running-tests-with-docker-compose)
+   - [Code Coverage](#code-coverage)
+
+
+# Background
+
+## Purpose and Scope for Clinical Trial Randomization
```

Comment:
It's a nice write-up, but we have already a brief intro about randomization. Maybe we could just redirect people to those nice articles? Maybe instead we could talk more about the project purpose (which is: to provide eCRFs with randomization API)